src/eq.js

```diff
@@ -0,0 +1,138 @@
+define(function (require) {
+  'use strict';
+
+  var p5Sound = require('master');
+  var Effect = require('effect');
+  var Filter = require('filter');
+
+  /**
+   * p5.EQ is an audio effect that performs the function of a multiband
+   * audio equalizer. Equalization is used to adjust the balance of 
+   * frequency compoenents of an audio signal. This process is commonly used
+   * in sound production and recording to change the waveform before it reaches
+   * a sound output device. EQ can also be used as an audio effect to create
+   * interesting distortions by filtering out parts of the spectrum. p5.EQ is
+   * built using a chain of Web Audio Biquad Filter Nodes and can be
+   * instantiated with 3 or 8 bands. Bands can be added or removed from
+   * the EQ by directly modifying p5.EQ.bands (the array that stores filters).
+   *  
+   * @class p5.EQ
+   * @constructor
+   * @param {Number} [_eqsize] [Constructor will accept 3 or 8, defaults to 3]
+   * @return {Object} [p5.EQ object]
+   *
+   * @example
+   * <div><code>
+   * 
+   * </code></div>
+   */
+  p5.EQ = function(_eqsize) {
+    Effect.call(this);
+
+    //p5.EQ can be of size (3) or (8), defaults to 3
+    _eqsize = _eqsize === 3 || _eqsize === 8 ? _eqsize : 3;
+
+    var factor;
+    _eqsize == 3 ? factor = Math.pow(2,3) : factor = 2;
+
+    //bands are stored in an array, index 0 - 3 or 0 - 7
+    this.bands = [];
```

Comment:
This needs documentation. "An array of modified p5.Filter objects," highlight their freq and gain methods, and provide a couple examples?